### PR TITLE
[BREAKING] Renamed the --julia CLI option to --yul

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: solc/CommandLineInterface.cpp [hunk 1]
```diff
@@ -785,13 +785,13 @@ bool CommandLineInterface::processInput()
...
 
-	if (m_args.count(g_argAssemble) || m_args.count(g_argStrictAssembly) || m_args.count(g_argJulia))
+	if (m_args.count(g_argAssemble) || m_args.count(g_argStrictAssembly) || m_args.count(g_argYul))
 	{
 		// switch to assembly mode
 		m_onlyAssemble = true;
 		using Input = AssemblyStack::Language;
 		using Machine = AssemblyStack::Machine;
-		Input inputLanguage = m_args.count(g_argJulia) ? Input::JULIA : (m_args.count(g_argStrictAssembly) ? Input::StrictAssembly : Input::Assembly);
+		Input inputLanguage = m_args.count(g_argYul) ? Input::JULIA : (m_args.count(g_argStrictAssembly) ? Input::StrictAssembly : Input::Assembly);
```
Comment (aj07) [inline solc/CommandLineInterface.cpp:790]: What about this one "Input::JULIA :"?

**Comment by chriseth** [comment]
Don't we want to keep it backwards compatible for some time? On the other hand, perhaps nobody acutally uses this in an automated way.

**Comment by axic** [comment]
I don't think anybody uses it and this is a breaking release so I'd just go with it.

**Comment by chriseth** [comment]
Just because it is a breaking release does not mean we _have_ to break everything ;-) Still fine with merging.

Please rebase!

**Comment by axic** [comment]
I agree, though on this I think we can make an exception :)

**Comment by axic** [comment]
Rebased.

**Review by chriseth — APPROVED** [review]

**Pull request merged**